.github/workflows/3x.yml

```diff
@@ -25,10 +22,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
       - name: Save repo content as artifact
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: full-workspace
           path: ${{ github.workspace }}
+          overwrite: true
```

Comment:
What motivates the new setting?  Did v3 always overwrite and now we need to explicitly say to do so for v4?

Reply:
I changed it because I was getting a resource contention in the build, but it seems that was happening because of the coverage save which has been removed. Removed this.